test/integration/util/util.go

```diff
@@ -92,15 +94,36 @@ func NewKubectlRunner(t *testing.T) *KubectlRunner {
 }
 
 func (k *KubectlRunner) RunCommand(args []string, outputObj interface{}) error {
-	args = append(args, "-o=json")
-	cmd := exec.Command(k.BinaryPath, args...)
-	output, err := cmd.CombinedOutput()
-	if err != nil {
-		return err
-	}
-	d := json.NewDecoder(bytes.NewReader(output))
+	args = append(args, "-o=json") //TODO: aprindle: ask Dan about this merging with RunCommandSimple
+	output := k.RunCommandSimple(args)
+	d := json.NewDecoder(bytes.NewReader([]byte(output)))
 	if err := d.Decode(outputObj); err != nil {
 		return err
 	}
 	return nil
 }
+
+func (k *KubectlRunner) RunCommandSimple(args []string) string {
+	cmd := exec.Command(k.BinaryPath, args...)
+	stdout, err := cmd.CombinedOutput()
+	if err != nil {
+		k.T.Fatalf("Error running command: %s %s. Error: %s, Output: %s", k.BinaryPath, args, err, stdout)
+	}
+	return string(stdout)
```

Comment:
Probably best to just return the byte slice, then RunCommandParseOutput can avoid casting back. Callers of RunCommandSimple can cast.